editions/tw5.com/tiddlers/community/Translate TiddlyWiki into your language.tid

```diff
@@ -1,12 +1,12 @@
 created: 20141126153016142
-modified: 20141126153016142
+modified: 20160602025338387
 tags: Community
 title: Translate TiddlyWiki into your language
 type: text/vnd.tiddlywiki
 
 There is a special edition of TiddlyWiki that simplifies creating and maintaining translations:
 
-http://tiddlywiki.com/editions/translators/
+http://tiddlywiki.com/prerelease/editions/translators/
```

Comment:
Perhaps here it would be better to link explicitly to the two versions:

* http://tiddlywiki.com/editions/translators/ (current version)
* http://tiddlywiki.com/prerelease/editions/translators/ (latest prerelease)